### ISSUE TITLE
Missing .js and .css files

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/hap#14]
Using ReleaseTools uploads the release archive to GitHUb, and then breaks with an error:

    .../tmp/hap-1.22/doc/*.js: No such file or directory

- either these files will be added after @grahamknockillaree's update of the HAP manual, or the script could allow to not to have them.

Anyway, the first goal is to be able to publish release archives on GitHUb, and that almost works (would be still useful to resolve #12, since makedoc.g takes much more time when invoked from ReleaseTools in comparison with `gap makedoc.g`).

**Comment by alex-konovalov** [comment]
.css files are also missing - that triggers an error while using ReleaseTools, see https://github.com/gap-packages/hap/wiki/HAP-wiki

**Comment by grahamknockillaree** [comment]
I've added the manual.css file.  It keeps disappearing from the github repository!!

I'm not sure what .js files are supposed to be here. I don't think I use any.

**Comment by alex-konovalov** [comment]
These files are ignored by https://github.com/gap-packages/hap/blob/master/.gitignore because they should not be kept under version control. Instead, they are added by GAPDoc when you build the manual - provided some modern GAPDoc call format is used. I suspect your GAPDoc invocation needs to be updated, following examples in some modern packages, but had no opportunity to check it.

**Comment by grahamknockillaree** [comment]
One step forward, two steps back! :)

On a quick (maybe too quick) first inspection it seems that AutoDoc insists on building the manual in the Hap1.24/doc directory. I've been using MakeGAPDocDoc() to build a manual describing functions in Hap1.24/doc, and then using MakeGAPDocDoc() a second time to build a manual describing examples in Hap1.24/doc/tutorial. Maybe I'm just too tired, but I don't see an option for doing this with AutoDoc.

**Comment by alex-konovalov** [comment]
Confused why AutoDoc appeared here - that's a step too far. Look at some package which uses GAPDoc, but not AutoDoc. For example, https://github.com/gap-packages/laguna/blob/master/makedoc.g in LAGUNA - it has 
```
# Copy the *.css and *.js files from the styles directory of the GAPDoc 
# package into the directory containing the package manual.
CopyHTMLStyleFiles( "doc" );
```
and I think this is what's missing in HAP.

**Comment by grahamknockillaree** [comment]
Thanks.  AutoDoc works fine and easily for building things in doc, but maybe not in doc/tutorial.  Tomorrow I'll look at Laguna.

**Comment by grahamknockillaree** [comment]
CopyHTMLStyleFiles( "doc" ); did the trick.  

So gap makedoc.g should work now.

**Comment by alex-konovalov** [comment]
Resolved by bd38b039f3ffc5f8bf5cab20ee1ff2e2355aed73
